addons/point_of_sale/models/pos_session.py

```diff
@@ -277,6 +278,21 @@ def action_pos_session_closing_control(self, balancing_account=False, amount_to_
             session.write({'state': 'closing_control', 'stop_at': fields.Datetime.now()})
             if not session.config_id.cash_control:
                 return session.action_pos_session_close(balancing_account, amount_to_balance)
+            # If the session is in rescue, we only compute the payments in the cash register
+            # It is not yet possible to close a rescue session through the front end, see `close_session_from_ui`
+            if session.rescue and session.config_id.cash_control:
+                default_cash_payment_method_id = self.payment_method_ids.filtered(lambda pm: pm.type == 'cash')[0]
+                orders = self.order_ids.filtered(lambda o: o.state == 'paid' or o.state == 'invoiced')
+                total_cash = sum(
+                    orders.payment_ids.filtered(lambda p: p.payment_method_id == default_cash_payment_method_id).mapped('amount')
+                ) + self.cash_register_balance_start
+                if total_cash > 0:
+                    self.env['account.bank.statement.cashbox'].create({
+                        'cashbox_lines_ids': [(0, 0, {'coin_value': total_cash, 'number': 1})],
```

Comment:
Do we really need this or not finally?

Reply:
because we can delete the model if we don't need it

Reply:
Well, by doing this there isn't any profit/loss for the rescue session and since the user can't open the pos ui to fill the cash details, it's better to assume that there's no profit/loss for the rescue no?

Reply:
putting lines in it and validating it is just like writing values on the bank statement "start" or "end" bank statement.

You can see it in "account_bank_statement.py" in accounting:
    def _validate_cashbox(self):
        for cashbox in self:
            if cashbox.start_bank_stmt_ids:
                cashbox.start_bank_stmt_ids.write({'balance_start': cashbox.total})
            if cashbox.end_bank_stmt_ids:
                cashbox.end_bank_stmt_ids.write({'balance_end_real': cashbox.total})

So you can probably just write on the statement like when we are putting the opening amount, like in "set_cashbox_pos" but on the end value.

Reply:
That would remove the need of cashbox and cashbox lines

Reply:
So we also need to change: post_closing_cash_details the same as here

Reply:
Changes were made regarding the use of the cashbox. Since time is a scarce resource, only the overridden model along with the views were removed. The original model (from the accounting module) should be removed at a later date.